### DISCUSSION TITLE
Well its finally "done"

### DISCUSSION THREAD

**Discussion by SeanTurner71** [rdudhagra/Sand-Table#23] [Show and tell]
I got derailed by another [project](https://www.youtube.com/watch?v=CuPCCtf7GqA&t=290s) and had a coworker build the table out of walnut and inlayed the glass which took time, but its in a very workable state.  The ball does not move the smoothest which I might work on but I'm not stressed about it.  Going to do video on it eventually but just happy to see it working.  Thanks for all the help, the programming really stressed me but I'm glad its done.
![20211023_134603](https://user-images.githubusercontent.com/86273998/138566585-13f7d4c8-83c6-4b7e-ab4d-f2b4aba90739.jpg)
![20211022_193116](https://user-images.githubusercontent.com/86273998/138566591-04a2885d-b842-4ffb-a491-ce003fae2714.jpg)
![20211023_125306](https://user-images.githubusercontent.com/86273998/138566600-f2d56f3e-e1ae-4142-9b82-42b5a96e3b1b.jpg)

**Comment by rdudhagra** [comment]
Woah this is sick @SeanTurner71 ! The final result looks fantastic and the craftsmanship of that table is *on-point*—glad I could be of some help!

**Comment by SeanTurner71** [comment]
https://www.youtube.com/watch?v=QCLmVpVkmQY
You might as well be the first one to see the video.

**Reply by rdudhagra** [reply]
Haha love the video…especially the timelapses at the end! Would you mind if I link to this on the repo’s `README`?

**Comment by SeanTurner71** [comment]
Not at all.  Go ahead.
Sean

**Comment by iwannabswiss** [comment]
@SeanTurner71 I watched your video and enjoyed the tracks.  I saw your software recommendations for making tracks, but I can't figure out how you made the butterfly version or the one at 7:00.  Could you help explain?  Thanks.

**Comment by iwannabswiss** [comment]
In case anyone else is interested.  I found the answers to my questions.  The butterfly version can be found [here](https://github.com/ddkengr/Sisyphus-threads), but I still am curious about how it was created.  The other track is called dither_hypnogrid, and that one is found [here](https://github.com/Dithermaster/sisyphus/tree/master/thr_paths).

**Reply by rdudhagra** [reply]
Thanks for posting those! The tracks that I use are from online sources like the ones you linked to. If you want to make your own though I've heard of folks using https://sandify.org and https://github.com/SlightlyLoony/JSisyphus

**Reply by iwannabswiss** [reply]
> Thanks for posting those! The tracks that I use are from online sources like the ones you linked to. If you want to make your own though I've heard of folks using https://sandify.org and https://github.com/SlightlyLoony/JSisyphus

You're welcome.  Since no one had responded and there isn't much discussion, I wanted to help make sure anyone who reads this in the future isn't left wondering.  

So far, I've used [Sandify](https://sandify.org/) and [Sand Table Pattern Maker](https://github.com/markroland/sand-table-pattern-maker), which are both excellent, BTW.  I've also made a few designs using [Sisyphus for the Rest of Us](https://sftrou.omarelamri.me/#/home); directions for how to use it can be found [here](https://www.youtube.com/watch?v=xWoSbuSkXeA&t=112s).  However, I'd love to expand my knowledge and abilities by taking them to the next level and designing tracks like those I had questions about.
